lib/storage/metadata/mongoclient/utils.js

```diff
@@ -5,15 +5,29 @@ const {
 } = require('../conditions');
 
 function escape(obj) {
-    return JSON.parse(JSON.stringify(obj).
-                      replace(/\$/g, '\uFF04').
-                      replace(/\./g, '\uFF0E'));
+    const _obj = {};
+    for (const prop in obj) {
+        if (Object.prototype.hasOwnProperty.call(obj, prop)) {
+            const _prop = prop.
+                replace(/\$/g, '\uFF04').
+                replace(/\./g, '\uFF0E');
+            _obj[_prop] = obj[prop];
+        }
+    }
+    return _obj;
```

Comment:
`Object.keys` would skip the inherited properties.
```suggestion
    const _obj = {};
    Object.keys(obj).forEach(prop => {
        const _prop = prop.
            replace(/\$/g, '\uFF04').
            replace(/\./g, '\uFF0E');
        _obj[_prop] = obj[prop];
    });
    
    return _obj;
```